llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -135,6 +135,38 @@ Counter CounterExpressionBuilder::subtract(Counter LHS, Counter RHS,
   return Simplify ? simplify(Cnt) : Cnt;
 }
 
+Counter CounterExpressionBuilder::replace(Counter C, const ReplaceMap &Map) {
+  auto I = Map.find(C);
+
+  // Replace C with the Map even if C is Expression.
+  if (I != Map.end())
+    return I->second;
+
+  // Traverse only Expression.
```

Comment:
Why?

Reply:
Because others (Zero or CounterRef) are not traversable :)

Is this line too naive to you? I could eliminate.